descriptors/DeploymentDescriptor-template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "srvcId": "${artifactId}-${version}",
```

Comment:
Please change to "srvcId": "@artifactId@-@version@",

Reply:
Done